src/controllers/notifyControllers.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable require-jsdoc */
+import models from '../models';
+
+const { notifications } = models;
+
+export default {
+
+  getAllNotification: async (req, res) => {
+    try {
+      const user_id = req.users.id;
+      const allRequests = await notifications.findAll({ where: { receiver: user_id } });
+      return res.status(200).json({ status: 200, data: allRequests });
```

Comment:
Thought we have a function that handles responses (`successResponseWithDatat` and `errorResppnse`). Why use `return res...`?